packages/cli/src/utils/deployer.ts

```diff
@@ -68,6 +68,7 @@ export const createTar = (path: string): Readable => {
   const mesgignore = join(path, '.mesgignore')
   const ig = ignore().add([
     '.git',
+    '.mesg',
```

Comment:
now the local dev chain is in the current directory so it might happen often that the current directory is also the directory of the service we are developing